modules/apps/object/object-test/src/testFunctional/macros/Picklist.macro

```diff
@@ -224,7 +224,7 @@ definition {
 
 		SelectFrame(locator1 = "Picklist#IFRAME_PICKLIST_ITEM_MODAL");
 
-		AssertElementPresent(locator1 = "Picklist#PICKLIST_ITEM_DETAILS");
+		AssertElementPresent(locator1 = "Picklist#ADD_PICKLIST_ITEM_NAME");
```

Comment:
Your fix makes this test work again. But I think we can improve this assertion.
This test is asserting that the field element is present at the page, but it's not checking its value. This might lead to bug don't be cached, for example, if the input is being exhibited empty or with the wrong value.
What you could to is to use the function bellow to assert the value it's actually there:
```
AssertValue(
	key_text = "Key",
	locator1 = "TextInput#ANY",
	value1 = "open");
```
What do you think of changing the assert this way? Let me you if you have a different idea or have any questions.